src/soloud/src/core/soloud.cpp

```diff
@@ -205,7 +205,24 @@ namespace SoLoud
 		mAudioThreadMutex = NULL;
 	}
 
-	result Soloud::init(unsigned int aFlags, unsigned int aBackend, unsigned int aSamplerate, unsigned int aBufferSize, unsigned int aChannels)
+	// Change output device.
+	// Added by Marco Bavagnoli
+	result Soloud::miniaudio_changeDevice(void *pPlaybackInfos_id)
+	{
+#if defined(WITH_MINIAUDIO)
+		// lockAudioMutex_internal();
```

Comment:
Should these mutex locks be commented out?

Reply:
removed